netmiko/help.md

```diff
@@ -98,6 +101,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.0.2 - Updated python libraries
```

Comment:
1.0.1 hasn't been released https://market.komand.com/plugins/rapid7/netmiko/1.0.0

So set version to 1.0.1, regenerate, and combine the changelog lines.